docs/contributing/index.md

```diff
@@ -6,7 +6,7 @@ with the community and maintainers via
 [a GitHub Discussion](https://github.com/nsidc/earthaccess/discussions),
 or [any other method](our-meet-ups.md).
 
-Please note that we have a [code of conduct](./code-of-conduct.md). Please follow it in all of your interactions with the project.
+Please note that we have a [code of conduct](/CODE_OF_CONDUCT.md). Please follow it in all of your interactions with the project.
```

Comment:
This 404s, I think we need to change it back :)

Here's the link it generates in the PR preview: https://earthaccess--744.org.readthedocs.build/CODE_OF_CONDUCT.md